### PR TITLE
Glossary update for 14.0

### PR DESCRIPTION
**Pull request by corinnaguenther**
[#49185] Keep Glossary updated

https://community.openproject.org/work_packages/49185

### CODE REVIEW

**Comment by corinnaguenther** [comment]
Hi @cbliard, do you think you could review the (small) paragraph about progress reporting (defining Work/Remaining Work and % Complete) that I updated for our Glossary?

File: docs/glossary/README.md [hunk 1]
```diff
@@ -284,6 +284,10 @@ In OpenProject, a [project](../user-guide/projects/) is defined as an individual
 - an overarching team working on one topic or
 - separate products or customers.
 
+### Project attribute
+
+A project attribute in OpenProject is a [custom field](#custom-field) that applies to an entire project. It can be displayed in the right-hand column on the project overview page and can be divided into sections for a better structure.
```
Comment (MayaBerd) [inline docs/glossary/README.md]:
```suggestion
A project attribute in OpenProject is a [custom field](#custom-field) that applies to an entire project. It is displayed in the right-hand column on the project overview page and can be divided into sections for a better structure.
```

**Review by MayaBerd — APPROVED** [review]

File: docs/glossary/README.md [hunk 2]
```diff
@@ -396,9 +400,12 @@ In OpenProject, a wiki is defined as a [module](#module) that allows to use wiki
 
 ![A wiki module in OpenProject](glossary-openproject-wiki.png)
 
-### Work and Remaining Work
+### Work, Remaining Work and % Complete
+
+In OpenProject, '**Work**' refers to a work package attribute indicating the estimated hours needed to complete a task.  
+'**Remaining work**' is a work package attribute that shows how much work is left to finish the work package. It is automatically calculated if you work with status-based progress reporting. And '**% Complete**'shows an automatically calculated work package attribute that shows in percentage how much work is already completed. All three attributes are important for [progress reporting with OpenProject](https://www.openproject.org/blog/changes-progress-work-estimates/).
```
Comment (cbliard) [inline docs/glossary/README.md]: I would add a link for "status-based progress reporting" to https://www.openproject.org/docs/user-guide/time-and-costs/progress-tracking/#status-based-progress-reporting

And `And '**% Complete**'shows an automatically calculated work package attribute that shows in percentage ...` feels wrong. What about `And '**% Complete**' is an automatically calculated work package attribute that shows in percentage ...`?

Comment (cbliard) [reply docs/glossary/README.md]: Maybe "It is automatically calculated if you work with status-based progress reporting." could even be removed. It's a detail and it is better explained in "progress reporting with OpenProject" page.

**Review by cbliard — APPROVED** [review]
Everything is correct. I only have a minor remark.

File: docs/glossary/README.md (hunk 2 above)
Comment (corinnaguenther) [reply docs/glossary/README.md]: Thanks a lot!

**Pull request merged**